doc/classes/Mesh.xml

```diff
@@ -153,12 +153,6 @@
 				[b]Note:[/b] This is only implemented for [ArrayMesh] and [PrimitiveMesh].
 			</description>
 		</method>
-		<method name="get_faces" qualifiers="const">
-			<return type="PackedVector3Array" />
-			<description>
-				Returns all the vertices that make up the faces of the mesh. Each three vertices represent one triangle.
```

Comment:
This moved to ArrayMesh, so you should copy the description there.

Reply:
So this moved back to `Mesh` it seems, but the docs were still lost in the process.